pkg/telemetry/mocks/mock_tracing_server.go

```diff
@@ -0,0 +1,48 @@
+package mocks
```

Comment:
Let's not put mocks in the public `pkg` package. That opens them up to a public contract with other Go devs importing the OpenFGA library.

Reply:
Moved all our existing mocks to `internal/mocks`